### DISCUSSION TITLE
[Choix du statut] Calcul du taux de cotisations sociales

### DISCUSSION THREAD

**Discussion by glopezrios** [betagouv/mon-entreprise#2957] [General]
Dans les résultats apportés par les simulateurs "choix du statut" et "comparateur de statut", le taux de cotisations sociales est calculé en fonction de la "[rémunération totale](https://mon-entreprise.urssaf.fr/documentation/dirigeant/r%C3%A9mun%C3%A9ration/totale)", c'est-à-dire le chiffres d'affaires moins les charges (hors rémunération dirigeant).

Or cette façon de calculer le % de cotisations sociales n'est pas précisée. Cela déboussole plusieurs usagers qui croient que ce taux est mal calculé. Par exemple:

- pour l'auto-entrepreneur, il y a confusion entre ce taux de cotisations sociales et celui effectivement prélevé sur le chiffre d'affaires

- pour le travailleur indépendant, de nombreux sites indique que les cotisations sociales sont de l'ordre de 45%. Mais ce pourcentage régulièrement cité est calculé en fonction du [revenu imposable](https://mon-entreprise.urssaf.fr/documentation/imp%C3%B4t/revenu-imposable), et non en fonction de la "[rémunération totale](https://mon-entreprise.urssaf.fr/documentation/dirigeant/r%C3%A9mun%C3%A9ration/totale)". Autrement dit, le pourcentage de cotisations affiché dans le simulateur ne tient pas compte du montant des cotisations sociales.

Dans une logique de comparaison, il est essentiel que le taux de cotisations sociale soit calculé de la même façon pour les différents statuts (ce qui est le cas dans les simulateurs), mais il faudrait peut-être préciser la manière de le calculer.

**Comment by johangirod** [comment]
Oui, c'était un choix délibéré de parler de cotisation par rapport au chiffre d'affaire (ou bénéfice), plutôt que par rapport au revenu net, comme affiché d'habitude : 
- Cela permettait d'être cohérent par rapport à l'auto-entrepreneur
- Nous avions remarqué que le chiffre de 45% n'était pas clair pour beaucoup de monde. Des retours de tests usager nous laissait comprendre que la beaucoup pensaient que le chiffre était calculé par rapport au total (« l'Urssaf me prend 45% de ce que je gagne !!! »). Nous avons donc délibérément choisi ce rapport pour contrer cette idée reçue.

En revanche, je suis d'accord que le chiffre mérite d'être explicité. Peut-être tout simplement en reprenant le petit graphe de répartition existant sur les autres simulateurs ? 

![image](https://github.com/betagouv/mon-entreprise/assets/1775934/ca91ed5a-7af5-49f0-8db4-f43b15ef17d0)

**Comment by glopezrios** [comment]
A prolonger dans #3016
